### PR TITLE
kevin/ENG-1026

### PR DESCRIPTION
**Pull request by kcvan**
## Description
Why did you write this code?
add footer and header to /work
## How to Test
- [ ] Visit /work
- [ ] Verify that the footer is below the viewport and works on mobile/tablet/desktop
- [ ] Verify that the header shows and is fixed to the top
- [ ] Login and verify that the header changes to authenticated state
- [ ] Logout and verify that the header changes to unauthenticated state
- [ ] Verify that mobile view works for the above

Which devices did you test on?
- [x] Chrome on Mac
- [ ] Chrome on PC
- [ ] Firefox on Mac
- [ ] Firefox on PC
- [ ] Safari iPhone
- [ ] Chrome Android

### CODE REVIEW

**Comment by kcvan** [comment]
todo: create header

File: src/components/work/Footer/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+          </svg>
+        </Col>
+
+        {contentData.map(item => (
+          <Col sm="3" lg="2" className="mb-4 mb-lg-0" key={item.header}>
+            <ListGroup className="list-group-borderless" flush>
```
Comment (kcvan) [inline src/components/work/Footer/Footer.tsx]: might make this collapsable in the future, currently matches the `front` theme

File: src/components/work/Footer/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+              San Francisco, CA 94103
+            </ListGroupItem>
+          </ListGroup>
+        </Col>
+      </Row>
+    </Container>
+
+    <Container className="space-1 d-flex flex-column flex-md-row justify-content-sm-between align-items-center align-items-md-end">
+      <div className="d-flex d-md-none mb-3">
+        <svg width="190px" height="36px" viewBox="0 0 190 36" fill="#377dff">
```
Comment (tc) [inline src/components/work/Footer/Footer.tsx]: what is this SVG? can it be a separate component?

File: src/components/work/Footer/Footer.tsx [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
...28,31.7780016 L62.311828,5.14285714 Z M69.3973237,28.1168569 C70.36229,28.6909621 71.4525059,28.9778334 72.6676084,28.9778334 C73.8827108,28.9778334 74.9729267,28.6909621 75.93753,28.1168569 C76.9032224,27.5423891 77.6536307,26.7408904 78.1905702,25.7112728 C78.7260576,24.6831059 78.9947088,23.5214767 78.9947088,22.2292867 C78.9947088,20.9613955 78.7202489,19.8128224 78.1720551,18.7832049 C77.6234982,17.7546753 76.8734529,16.9520885 75.920467,16.378346 C74.966755,15.8035155 73.8827108,15.5162816 72.6676084,15.5162816 C71.4525059,15.5162816 70.36229,15.8035155 69.3973237,16.378346 C68.4319943,16.9520885 67.6757773,17.7546753 67.1275835,18.7832049 C66.5790267,19.8128224 66.3049298,20.9613955 66.3049298,22.2292867 C66.3049298,23.5457755 66.5790267,24.7128447 67.1275835,25.7290436 C67.6757773,26.7470557 68.4319943,27.5423891 69.3973237,28.1168569 Z M10.2150538,18.2529086 L31.155914,6.17142857 L52.0967742,18.2529086 L52.0967742,36 L31.155914,23.9192498 L31.155914,12.2121686 L15.4502688,21.2732786 L15.4502688,32.9807247 L10.2150538,36 L10.2150538,18.2529086 Z M0,11.7271867 L20.4568916,0 L25.5376344,2.93294846 L5.11440194,14.6583632 L5.10938888,31.8857143 L0,28.9545378 L0,11.7271867 Z" />
```
Comment (tc) [inline src/components/work/Footer/Footer.tsx]: same here, can we make this svg a component

Comment (kcvan) [reply src/components/work/Footer/Footer.tsx]: 👍

File: src/components/work/Footer/Footer.tsx (hunk 2 above)
Comment (kcvan) [reply src/components/work/Footer/Footer.tsx]: 👍

File: src/components/work/Footer/Footer.tsx [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+    btnClass: 'btn-github',
+    iconClass: 'fa-github',
+  },
+  {
+    btnClass: 'btn-linkedin',
+    iconClass: 'fa-linkedin',
+  },
+];
+
+const BeenestSVG = () => (
```
Comment (tc) [inline src/components/work/Footer/Footer.tsx]: will this be re-used in the header too?

Comment (kcvan) [reply src/components/work/Footer/Footer.tsx]: yes, moving this to a shared folder

File: src/components/work/Header/Header.tsx [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+    <header className="fixed-top bg-white" id="bee-main-header">
+      <Navbar light expand="md">
+        <NavbarBrand href="/work">
+          <BeenestSVGPrimary />
+        </NavbarBrand>
+        <NavbarToggler onClick={handleToggleNavbar} className="mr-2" />
+        <Collapse isOpen={isOpen} navbar>
+          <Nav className="ml-auto u-header__navbar-nav pt-5 pt-md-0" navbar>
+            <Button type="button" outline color="primary" className="mb-4 mb-md-0 mr-md-4">
+              Become a Host
```
Comment (tc) [inline src/components/work/Header/Header.tsx]: href to `/hosts/signup`

Comment (kcvan) [reply src/components/work/Header/Header.tsx]: 👍

**Pull request merged**